spec/controllers/api/submissions_controller_spec.rb

```diff
@@ -20,6 +22,13 @@
         it { is_expected.to include 'message' }
         it { is_expected.to include 'result' }
       end
+
+      describe 'when sent invalid data from the public' do
```

Comment:
Agreed, happy to discuss testing strategy, as per [pivotal](https://www.pivotaltracker.com/story/show/112878143)